releasenotes/notes/fix-add-b3-traceid-and-spanid-padding-919b698c8008346d.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    tracing: This fix resolves an edge case where Datadog trace_id and span_ids that were not standard length
```

Comment:
Can we simplify this to just state the customer impact. ex: `tracing: Pads trace_id and span_ids in b3 headers to have a minimum length of 16"

Reply:
Sure, I was just following the suggested format but happy to shorten it